resources/com/github/coreos/job.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Couldn't we fold this into the `buildconfig.json` template?

Reply:
To be honestly I tried. But I couldn't find an easier way to do it. Probably due the lack of experiencie with ocp. ='( I think we would need to find a way for the Job to create the bc, and the imageStream or to trigger the build, so it would get the ownerReference. Probably we would need an image with the oc installed and some way for it to access the api inside it?
The way I found was to create the Job first and then add the ownerReference in the bc and imageStream.

Reply:
Ahh right OK, that makes total sense now why you had to do instantiate them separately!